Documentation/install/upgrade.rst

```diff
@@ -178,6 +180,42 @@ Annotations:
    upgrade. Connections should successfully re-establish without requiring
    clients to reconnect.
 
+1.3 Upgrade Notes
+-----------------
+
+Upgrading from 1.2.x to 1.3.y
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. If you are running a 1.2.x release, skip this step. If you are running
```

Comment:
IMO it's a little more clear if this logic is reversed, ie:

`If you are running a 1.1.x or earlier release, please upgrade to 1.2.x first...`